app-typescript/components/avatar/avatar-image.tsx

```diff
@@ -3,22 +3,26 @@ import {IPropsBase} from './interfaces';
 
 interface IPropsImageAvatar extends IPropsBase {
     imageUrl?: string | null; // defaults to a placeholder image
+    onClick?(): void;
 }
 
 export class AvatarContentImage extends React.PureComponent<IPropsImageAvatar> {
     render() {
-        if (this.props.imageUrl == null) {
+        const {imageUrl, tooltipText, onClick} = this.props;
+        const role: string | undefined = onClick == null ? undefined : 'button';
+
+        if (imageUrl == null) {
             return (
-                <span className="sd-avatar-content sd-avatar-content--dummy-img" title={this.props.tooltipText}>
+                <span className="sd-avatar-content sd-avatar-content--dummy-img" title={tooltipText} role={role}>
```

Comment:
@tomaskikutis Where is the `onClick` property assigned?

Reply:
good catch, somehow missed that. Fixing in https://github.com/superdesk/superdesk-ui-framework/pull/801